### PR TITLE
Remove the white phosphorous recipe added in #74599

### PR DESCRIPTION
**Pull request by kevingranade**
It's possible to do but is much more involved than this recipe makes it

#### Summary
None

#### Purpose of change
On review of the source https://en.wikipedia.org/wiki/Phosphorus#Bone_ash_and_guano used in #74599 the recipe added doesn't remotely resemble the actual recipe.

"Bone ash was the major source of phosphorus until the 1840s. The method started by roasting bones, then employed the use of [fire clay](https://en.wikipedia.org/wiki/Fire_clay) [retorts](https://en.wikipedia.org/wiki/Retorts) encased in a very hot brick furnace to distill out the highly toxic elemental phosphorus product.[[54]](https://en.wikipedia.org/wiki/Phosphorus#cite_note-54) Alternately, precipitated phosphates could be made from ground-up bones that had been de-greased and treated with strong acids. White phosphorus could then be made by heating the precipitated phosphates, mixed with ground coal or [charcoal](https://en.wikipedia.org/wiki/Charcoal) in an iron pot, and distilling off phosphorus vapour in a [retort](https://en.wikipedia.org/wiki/Retort).[[55]](https://en.wikipedia.org/wiki/Phosphorus#cite_note-threlfall-v-55) [Carbon monoxide](https://en.wikipedia.org/wiki/Carbon_monoxide) and other flammable gases produced during the reduction process were burnt off in a [flare stack](https://en.wikipedia.org/wiki/Gas_flare)."

Basically you need to process the bones in a kiln to produce elemental phosphorous, or react them with strong acid to create a phosphorous precipitate which is then reacted with charcoal and distilled to yield phosphorous.

#### Describe the solution
Just removes the recipe.

#### Describe alternatives you've considered
I don't have time to dig into the proper recipe other than briefly describing it here.

### CODE REVIEW

**Comment by Rocket-F-1024** [comment]
Maybe add a recipe for heating the bone meal or using strong acids to acidify the bone meal to produce calcium phosphate, and then use the calcium phosphate to produce white phosphorus in the same recipe as before, what do you think? 🤔

**Comment by kevingranade** [comment]
Yes go ahead.

**Comment by zachary-kaelan** [comment]
https://www.militarynewbie.com/wp-content/uploads/2013/11/TM-21-210-Improvised-Munitions-Handbook-1969-Department-of-the-Army.pdf

Page 173 has an improvised white flare and page 184 has an improvised white smoke munition. Might be more practical than white phosphorus.

**Comment by Holli-Git** [comment]
> https://www.militarynewbie.com/wp-content/uploads/2013/11/TM-21-210-Improvised-Munitions-Handbook-1969-Department-of-the-Army.pdf
> 
> Page 173 has an improvised white flare and page 184 has an improvised white smoke munition. Might be more practical than white phosphorus.

I don't think this is really relevant to the PR past "here's a use for WP"
